lib/Doctrine/DBAL/Schema/PostgreSqlSchemaManager.php

```diff
@@ -154,11 +154,12 @@ protected function _getPortableTableForeignKeyDefinition($tableForeignKey)
             $onDelete = $match[1];
         }
 
-        assert(preg_match(
+        $match = preg_match(
```

Comment:
This executes the regex now, though, which shouldn't be done - according to the previous code.